src/Symfony/Component/DependencyInjection/Loader/FileLoader.php

```diff
@@ -122,6 +123,13 @@ public function registerClasses(Definition $prototype, string $namespace, string
         $serializedPrototype = serialize($prototype);
 
         foreach ($classes as $class => $errorMessage) {
+            if (null === $errorMessage && $autoconfigureAttributes) {
+                $r = $this->container->getReflectionClass($class);
```

Comment:
$r is created 3 times in the loop, let's move it before the if and reuse it?

Reply:
I factorize a bit the implementation, but not totally because the class might not exist